src/util/XDRCereal.h

```diff
@@ -97,6 +97,14 @@ cereal_override(cereal::JSONOutputArchive& ar, const stellar::Asset& s,
     xdr::archive(ar, stellar::assetToString(s), field);
 }
 
+template <typename T>
+typename std::enable_if<xdr::xdr_traits<T>::is_enum>::type
+cereal_override(cereal::JSONOutputArchive& ar, const T& t, const char* field)
```

Comment:
One thing I'm wondering is whether we should make it optional to print names (for human readable purposes) or values (for computer purposes). Another option would be to always print names and optionally print values. Perhaps this behavior could be controlled by `--compact`. @MonsieurNicolas where do you stand on this issue?

Reply:
I thought about it after our conversation, but I left it out for now because the current code seems to only print human readable names (both `print-xdr` and `dump-xdr`).

Reply:
Yeah we can keep just friendly names for now.
If we need automation to do something with those I suspect that they can just treat them as strings and are very unlikely to need to convert it back to binary. If we ever need this, we can consider adding a new option when we need this.